wandb/sdk/internal/handler.py

```diff
@@ -592,7 +598,11 @@ def handle_tbrecord(self, record: Record) -> None:
         logger.info("handling tbrecord: %s", record)
         if self._tb_watcher:
             tbrecord = record.tbrecord
-            self._tb_watcher.add(tbrecord.log_dir, tbrecord.save, tbrecord.root_dir)
+            if tbrecord.log_type == 1 and self._prof_watcher:
```

Comment:
Use `pb.TBRecord.LogType.PROFILER`
and the else should check pb.TBRecord.LogType.TFEvents

which will work for old files because protobuf defaults to zero